compiler/vm.nim

```diff
@@ -1644,10 +1644,22 @@ proc rawExecute(c: PCtx, start: int, tos: PStackFrame): TFullReg =
     of opcEqIdent:
       decodeBC(rkInt)
       # aliases for shorter and easier to understand code below
-      let aNode = regs[rb].node
-      let bNode = regs[rc].node
-      # these are cstring to prevent string copy, and cmpIgnoreStyle from
-      # takes cstring arguments
+      var aNode = regs[rb].node
+      var bNode = regs[rc].node
+      # Skipping both, `nkPostfix` and `nkAccQuoted` for both
+      # arguments.  `nkPostfix` exists only to tag exported symbols
+      # and therefor it can be safely skipped. Nim has no postfix
+      # operator. `nkAccQuoted` is used to quote an identifier that
+      # wouldn't be allowed to use in an unquoted context.
```

Comment:
Bah, everybody who touches this code should really know this already...

Reply:
Yeah I am, because it's noise here. The AST is documented well enough and compiler hackers start as macro hackers.

Reply:
I am thinking of future maintainers of this codebase. If somebody in the future, when I am not active anymore, decides for unknown reasons to use the `nkPostfix` also for something else that isn't  marking a procedure as exported, this comment will tell the future maintainers of this code that the reasons for skipping the ``nkPosfix`` node aren't true anymore and this code needs to be adopted. These types of comments that explain **why** things are done in a specific way are what I am missing a lot in the Nim codebase. Those questions are not answered just studying the code.

Reply:
Silly nonsense.